File: app/src/main/java/com/weathercock/cleanarchitecture/domain/model/weather/TemperatureUnitModel.java
package com.weathercock.cleanarchitecture.domain.model.weather;

/**
 * Temperature Unit Model
 * <p>
 * Created by tamura_k on 2016/06/03.
 */
public class TemperatureUnitModel {

    //摂氏
    private Float celsius;
    //華氏
    private Float fahrenheit;

    public Float getCelsius() {
        return celsius;
    }

    public void setCelsius(Float celsius) {
        this.celsius = celsius;
    }

    public Float getFahrenheit() {
        return fahrenheit;
    }

    public void setFahrenheit(Float fahrenheit) {
        this.fahrenheit = fahrenheit;
    }

}

File: app/src/main/java/com/kazakago/cleanarchitecture/domain/model/weather/TemperatureUnitModel.kt
package com.kazakago.cleanarchitecture.domain.model.weather

import paperparcel.PaperParcel
import paperparcel.PaperParcelable

@PaperParcel
data class TemperatureUnitModel(
        //摂氏
        val celsius: Float,
        //華氏
        val fahrenheit: Float
) : PaperParcelable {

    companion object {
        @JvmField
        val CREATOR = PaperParcelTemperatureUnitModel.CREATOR
    }

}
